kratos/python_scripts/run_tests.py

```diff
@@ -87,8 +88,10 @@ def RunTestSuitInTime(self, application, applicationPath, path, level, verbose,
                 self.process.terminate()
                 t.join()
                 print('\nABORT: Tests for {} took to long. Process Killed.'.format(application), file=sys.stderr)
+                return 1
             else:
                 print('\nTests for {} finished in time ({}s).'.format(application, timeout))
+                return
```

Comment:
Actually it should not be returning at all, but the return above should change the exitCode from the commander. Working on it